test/integration/publishing/publish-calculations.test.js

```diff
@@ -122,4 +122,11 @@ describe('publish calculations', () => {
     await publish()
     expect(mockSendMessage.mock.calls[0][0].body.fundings.length).toBe(2)
   })
```

Comment:
consider changing description of previous test to something like `should not publish same calculation on second run if record has not been updated after previous run`